src/karts/controller/test_ai.cpp

```diff
@@ -315,7 +315,7 @@ void SkiddingAI::update(int ticks)
     computeNearestKarts();
 
     m_kart->setSlowdown(MaxSpeed::MS_DECREASE_AI,
-                        m_ai_properties->getSpeedCap(m_distance_to_player),
+                        m_ai_properties->getSpeedCap(m_distance_to_player, 2, 3),
```

Comment:
Why are 2,3 hardcoded?

Reply:
It returns the average of the first speed cap and the last speed cap. I could have put 1,1 as well, though, as the test_ai is only used in profile_mode where there will be no slowdown anyway.

Plus, the test_ai is supposed to be updated with the new skidding_ai once changes have been validated (it has not been done recently, but it should be), at which point this will be overwritten by the changes seen in skidding_ai.

Basically, it doesn't matter and the main motivation behind changing anything at all in this file is avoiding a compilation error.

Reply:
A little comment explaining that would be welcome. something like 1,1 /* N/A, no rubberband in profile mode */);